### PR TITLE
Fix statics

### PR DESCRIPTION
**Pull request by godu**

### CODE REVIEW

File: packages/@coorpacademy-app-review/package.json [hunk 1]
```diff
@@ -7,8 +7,7 @@
   },
   "license": "UNLICENSED",
   "scripts": {
-    "static": "webpack --progress --output-path static/dist",
-    "poststatic": "cp sandbox/index.html static/index.html",
```
Comment (godu) [inline packages/@coorpacademy-app-review/package.json:11]: Webpack is in charge to serve this file through `HtmlWebpackPlugin` plugin.

File: packages/@coorpacademy-app-review/webpack.config.js [hunk 2]
```diff
@@ -45,13 +44,15 @@ module.exports = pipe(
     concat([
       new webpack.DefinePlugin({
         'process.env.API_TEST_TOKEN': JSON.stringify(API_TEST_TOKEN)
+      }),
+      new HtmlWebpackPlugin({
```
Comment (godu) [inline packages/@coorpacademy-app-review/webpack.config.js:48]: Webpack is able to serve html files and inject js/css dependencies.

File: packages/@coorpacademy-nova-icons/package.json [hunk 3]
```diff
@@ -56,7 +56,7 @@
     "flow-bin": "^0.125.1",
     "flow-copy-source": "^2.0.9",
     "glob": "^7.1.6",
-    "mkdirp-promise": "^5.0.1",
```
Comment (godu) [inline packages/@coorpacademy-nova-icons/package.json:59]: `mkdirp-promise` is no longer maintained.

**Comment by chrisdugne** [comment]
bien joué ce `HtmlWebpackPlugin`, par contre je ne suis pas sur de la config prettier là dessus

**Review by chrisdugne — APPROVED** [review]

**Pull request merged**